### ISSUE TITLE
performance of pretrained model monodepth2 not match

### ISSUE THREAD

**Issue by xjixzz** [SeasonDepth/SeasonDepth#6]
I evaluated  monodepth2 using the model of  mono+stereo_640x192 pretrained on KITTI but found its performance is much better than the results in your paper.  My experiments are based the validation set (17225 images) of SeasonDepth and the results are as follows:
AbsRel Average: 0.2545
a1 Average: 0.6276
AbsRel Variance 10^(-2): 0.0307
a1 Variance 10^(-2): 0.0498
AbsRel Relative Range: 0.2340
a1 Relative Range: 0.2287

I did not finetune the model on the training set of SeasonDepth but got much better results. I also evaluated monodepth2 on the training dataset of SeasonDepth and got the similar results:
AbsRel Average: 0.2565
a1 Average: 0.6100
AbsRel Variance 10^(-2): 0.0728
a1 Variance 10^(-2): 0.1763
AbsRel Relative Range: 0.3398
a1 Relative Range: 0.3203

I also evaluated the  model of  mono+stereo_1024x320 pretrained on KITTI and still got better results than your paper. By the way, all my experiments are based your evaluation code.

It is so strange and I wonder what is wrong.

**Comment by yangbaoquan** [comment]
There was probably a mistake when we ran test code of monodepth2.
We will check our experiment carefully before giving you an explanation.
By the way, is it convenient for you to provide your predicted depth maps so we can check it more quickly?

**Comment by xjixzz** [comment]
my [predicticted  depth maps](https://drive.google.com/file/d/1EI68cnqYsF5nRLvLPKYo8_ejGFSVfyhl/view?usp=sharing) of validation set of SeasonDepth 
using [mono+stereo_640x192](https://storage.googleapis.com/niantic-lon-static/research/monodepth2/mono%2Bstereo_640x192.zip)
[results](https://drive.google.com/drive/folders/16gxXtdAPkC5y7PUYmoeMmnd3ZnlQCTU8?usp=sharing)

**Comment by yangbaoquan** [comment]
Thank you for your help.

We have rerun test for monodepth2 and find that we made a mistake when we firstly ran test two years ago.

Below are the correct results, which are very close to yours:

![6328f80f0df0a12da62791207374904](https://user-images.githubusercontent.com/79578919/157610733-1f62e051-4d49-4802-a76f-e50d5624f07c.png)

We are going to update the benchmark in readme soon.

My sincere appreciation again!
